lib/layout/list_view.dart

```diff
@@ -8,22 +9,17 @@ import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart' as flutter;
 
 // ignore: must_be_immutable
-class ListView extends BoxLayout {
+class ListView extends ListViewWidget {
```

Comment:
I didn't catch this before. Why have both ListView and ListViewWidget when they are not used by anything else? Please combine them